persistence/sql/persister_session.go

```diff
@@ -128,12 +135,23 @@ func (p *Persister) ListSessionsByIdentity(ctx context.Context, iID uuid.UUID, a
 	nid := p.NetworkID(ctx)
 
 	if err := p.Transaction(ctx, func(ctx context.Context, c *pop.Connection) error {
+		i, err := p.GetIdentity(ctx, iID)
```

Comment:
We're now no longer respecting the expand property and instead fetch the identity always. I don't think that's intended? Please move it back to where it was, but of course outside of the loop to prevent unneccessary queries

```

 		if expandables.Has(session.ExpandSessionIdentity) {
 			i, err := p.GetIdentity(ctx, iID)
 			// ...
 			for _, s := range s {

```